### PR TITLE
Remove IBFT consensus protocol

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
# Description

Remove all IBFT-related stuff.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### Manual tests

Please complete this section if you ran manual tests for this functionality, otherwise delete it

# Documentation update

Please link the documentation update PR in this section if it's present, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: command/common.go [hunk 1]
```diff
@@ -27,8 +18,11 @@ const (
 	ValidatorPrefixFlag   = "validators-prefix"
 	MinValidatorCountFlag = "min-validator-count"
 	MaxValidatorCountFlag = "max-validator-count"
+)
 
-	IBFTValidatorTypeFlag = "ibft-validator-type"
+var (
+	MinValidatorCount = uint64(1)
```
Comment (goran-ethernal) [inline command/common.go]: This should be 3 right? Polybft won't work with just 1 validator.

Comment (goran-ethernal) [reply command/common.go]: Also I would move this to `default.go` with the rest of the command defaults.

File: command/common.go [hunk 2]
```diff
@@ -27,8 +18,11 @@ const (
...
+	MaxValidatorCount = common.MaxSafeJSInt
```
Comment (goran-ethernal) [inline command/common.go]: Move to `default.go`.

File: command/genesis/genesis.go [hunk 3]
```diff
@@ -117,64 +113,43 @@ func setFlags(cmd *cobra.Command) {
...
-		cmd.Flags().Uint64Var(
-			&params.maxNumValidators,
-			command.MaxValidatorCountFlag,
-			common.MaxSafeJSInt,
-			"the maximum number of validators in the validator set for PoS",
-		)
+	cmd.Flags().Uint64Var(
+		&params.minNumValidators,
+		command.MinValidatorCountFlag,
+		1,
```
Comment (goran-ethernal) [inline command/genesis/genesis.go]:
```suggestion
		command.MinNumValidators,
```

File: e2e/framework/config.go [hunk 4]
```diff
@@ -33,34 +28,32 @@ type PredeployParams struct {
...
-	Signer                  crypto.TxSigner          // Signer used for transactions
-	MinValidatorCount       uint64                   // Min validator count
-	MaxValidatorCount       uint64                   // Max validator count
-	BlockTime               uint64                   // Minimum block generation time (in s)
-	IBFTBaseTimeout         uint64                   // Base Timeout in seconds for IBFT
+	JSONRPCPort             int             // The JSON RPC endpoint port
+	GRPCPort                int             // The GRPC endpoint port
+	LibP2PPort              int             // The Libp2p endpoint port
+	RootDir                 string          // The root directory for test environment
+	IBFTDirPrefix           string          // The prefix of data directory for IBFT
```
Comment (goran-ethernal) [inline e2e/framework/config.go]: Do we need this?

File: e2e/framework/config.go [hunk 5]
```diff
@@ -33,34 +28,32 @@ type PredeployParams struct {
...
+	IBFTDir                 string          // The name of data directory for IBFT
```
Comment (goran-ethernal) [inline e2e/framework/config.go]: Do we need this?

Comment (Stefan-Ethernal) [reply e2e/framework/config.go]: Removed https://github.com/Ethernal-Tech/blade/pull/1/commits/b141e0dfd2f7977ae56774b1340ef2d8b751b201

File: e2e/framework/config.go (hunk 4 above)
Comment (Stefan-Ethernal) [reply e2e/framework/config.go]: Removed https://github.com/Ethernal-Tech/blade/pull/1/commits/b141e0dfd2f7977ae56774b1340ef2d8b751b201

File: command/genesis/genesis.go (hunk 3 above)
Comment (Stefan-Ethernal) [reply command/genesis/genesis.go]: Fixed https://github.com/Ethernal-Tech/blade/pull/1/commits/b141e0dfd2f7977ae56774b1340ef2d8b751b201

File: command/common.go (hunk 2 above)
Comment (Stefan-Ethernal) [reply command/common.go]: Fixed https://github.com/Ethernal-Tech/blade/pull/1/commits/b141e0dfd2f7977ae56774b1340ef2d8b751b201

File: command/common.go (hunk 1 above)
Comment (Stefan-Ethernal) [reply command/common.go]: Fixed https://github.com/Ethernal-Tech/blade/pull/1/commits/b141e0dfd2f7977ae56774b1340ef2d8b751b201

**Review by goran-ethernal — DISMISSED** [review]

**Review by goran-ethernal — APPROVED** [review]

**Pull request merged**